server_test.go

```diff
@@ -0,0 +1,159 @@
+package bramble
+
+import (
+	"testing"
+
+	"github.com/movio/bramble/testsrv"
```

Comment:
High-five on the end-to-end tests! I was nervous about those being missing while updating test fixtures. Nice work!

Reply:
Thanks, yeah definitely more comfortable making changes in the planner/execution pipeline with real servers to check against